### ISSUE TITLE
revise make example

### ISSUE THREAD

**Issue by aaronpeikert** [aaronpeikert/reproducible-research#33]
```
manuscript.pdf: data/ìris_prepped manuscript.Rmd
     Rscript -e 'rmarkdown::render("manuscript.Rmd")'
data/ìris_prepped: R/prepare_data.R data/iris.csv
     Rscript -e 'source("R/prepare_data.R")'
```
Shouldn't it be `data/ìris_prepped.csv` instead of `data/ìris_prepped`?
Should we include:
`all: manuscript.pdf`
@brandmaier

**Comment by aaronpeikert** [comment]
cross ref #21

**Comment by brandmaier** [comment]
yes and yes

**Comment by brandmaier** [comment]
Closed by d75eb76.
